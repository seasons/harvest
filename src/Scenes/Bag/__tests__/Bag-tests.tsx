import React from "react"
import { Bag } from ".."
import renderer from "react-test-renderer"

// describe("Profile", () => {
//   it("renders properly", () => {
//     const component = renderer.create(<Profile />)
//     expect(component).toMatchSnapshot()
//   })
// })

describe("Bag", () => {
  it("renders properly", () => {
    const component = renderer.create(<Bag />)
    expect(component).toMatchSnapshot()
  })
})
