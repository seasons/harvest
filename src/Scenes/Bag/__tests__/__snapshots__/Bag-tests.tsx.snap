// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bag renders properly 1`] = `null`;
